src/components/calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import './calculator.css';
+
+const Calculator = () => {
+    return (
+    <div className="main">
+        <div className="answer">0</div>
+        <div className="firstbox">
+            <button className="button1-1">AC</button>
+            <button className="button1-2">+/-</button>
+            <button className="button1-3">%</button>
+            <button className="button1-4">÷</button>
+        </div>
+        <div className="secondbox">
+            <button className="button1-1">7</button>
+            <button className="button1-2">8</button>
+            <button className="button1-3">9</button>
+            <button className="button1-4">x</button>
+        </div>
+        <div className="thirdbox">
+            <button className="button1-1">4</button>
+            <button className="button1-2">5</button>
+            <button className="button1-3">6</button>
+            <button className="button1-4">-</button>
+        </div>
+        <div className="fourthbox">
+            <button className="button1-1">1</button>
+            <button className="button1-2">2</button>
+            <button className="button1-3">3</button>
+            <button className="button1-4">+</button>
+        </div>
```

Comment:
- [ ] As mentioned in the project requirements **Create at least 1 child component** Please make sure the calculator component has at least one child component